bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -571,10 +613,23 @@ protected void compactEntryLog(EntryLogMetadata entryLogMeta) {
         return entryLogMetaMap;
```

Comment:
This only gets called afaict with this.entryLogMetaMap.  I don't think it needs to be passed as a parameter or returned, particularly since removeEntryLogRecordSafely above assumes that it's using this.entryLogMetaMap.